employees/views_business_logic.py

```diff
@@ -0,0 +1,114 @@
+from employees.models import Employee
+
+
+class ContractExtension:
+
+
+    name=None
+    exp_date=None
+
+    def add_one_month(self, employee_id):
+        employee_to_update = Employee.objects.get(pk=employee_id)
+
+        self.name=employee_to_update.first_name+ ' '+employee_to_update.last_name
+
+
+        old_date = employee_to_update.contract_exp_date
+        self.exp_date=old_date
+
+        try:
+            day = old_date.day
+            old_month = old_date.month
+            old_year = old_date.year
+        except:
```

Comment:
Another nasty try/except block. We never should be force to use `try:` on assigning to variables.

Reply:
done.. good to have tests for this class, so i know it still works properly  :)